tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -156,7 +156,7 @@ def run_test(host, port, ssl_version, cipher, threshold):
 
     failed = 0
     tcpdump_filter = "dst port " + str(port)
-    tcpdump_cmd = ["sudo", "tcpdump", "-i", "lo", "-n", "-B", "65535", tcpdump_filter]
+    tcpdump_cmd = ["sudo", "tcpdump", "-l", "-i", "lo", "-n", "-B", "65535", tcpdump_filter]
```

Comment:
From the man page, so nobody else has to look it up:
```
-l
    Make stdout line buffered. Useful if you want to see the data while capturing it. E.g., 

            tcpdump -l | tee dat

    or 

            tcpdump -l > dat & tail -f dat

    Note that on Windows,``line buffered'' means ``unbuffered'', so that WinDump will write each character individually if -l is specified. 
    -U is similar to -l in its behavior, but it will cause output to be ``packet-buffered'', so that the output is written to stdout at the end of each packet rather than at the end of each line; this is buffered on all platforms, including Windows. 
```

https://www.tcpdump.org/manpages/tcpdump.1.html